core/services/pipeline/task.eth_abi_encode.go

```diff
@@ -53,9 +57,10 @@ func (t *ETHABIEncodeTask) Run(_ context.Context, vars Vars, inputs []Result) (r
 		if !exists {
 			return Result{Error: errors.Wrapf(ErrBadInput, "ETHABIEncode: argument '%v' is missing", arg.Name)}
 		}
-		val, err = convertToABIType(val, arg.Type)
+		logger.Warnf("  - %v: (%T / %v) %v", arg.Name, val, arg.Type.GetType(), val)
```

Comment:
This looks a little hard to grep for, is there a human component we could add?

Reply:
Whoops, didn't mean for this to be included, will remove